BusBoard.ConsoleApp/Program.cs

```diff
@@ -10,18 +11,36 @@ static void Main(string[] args)
     {
         ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
         var tflApiHelper = new TflApiHelper();
+        var postcodeApiHelper = new PostcodeApiHelper();
+
         while (true)
         {
-            Console.WriteLine("Welcome to the BusBoard please enter the bus stop code you wish to check or exit to exit");
+            Console.WriteLine("Welcome to the BusBoard please enter a postcode you wish to check or 'exit' to exit");
             var input = Console.ReadLine();
             if (input == "exit")
             {
                 Console.WriteLine("Thank you for using BusBoard");
                 break;
             }
-            var buses = tflApiHelper.GetBuses(input);
-            var topFive = buses.Take(5).Select(bus => bus.GetBusData());
-            Console.WriteLine(string.Join("\n", topFive));
+
+            var coordinate = postcodeApiHelper.GetPostcodeCoordinates(input);
+            var stopPoints = tflApiHelper.GetStopPoints(coordinate);
+            if (stopPoints == null)
+            {
+                Console.WriteLine("No nearby stops found for that postcode!");
+            }
+
+            else
+            {
+                var buses = new List<BusEntry>();
+                foreach (var stopPoint in stopPoints.Take(2))
```

Comment:
As mentioned before I wouldn't use a loop for just two items 🙂

Reply:
This one I thought could be useful as you may want to have the nearest x stops rather than just 2. I was thinking both this and the number of displayed buses could be a variable and the user could pick "Earliest X trains from nearest Y stops"

Reply:
I think this is a case of trying to future proof too much 🙂 You want to write the neatest code possible for the current requirements. You can always refactor it if the requirements change 🙂